doc/radosgw/metrics.rst

```diff
@@ -191,6 +191,75 @@ Cache sizing can depend on a number of factors. These factors include:
 
 To help calculate the Ceph Object Gateway's memory usage of a cache, it should be noted that each cache entry, encompassing all of the op metrics, is 1360 bytes. This is an estimate and subject to change if metrics are added or removed from the op metrics list.
 
+Lifecycle Metrics
+=================
+
+The following metrics related to lifecycle (LC) processing are tracked per bucket by the Ceph Object Gateway.
+
+.. list-table:: Ceph Object Gateway Lifecycle Metrics
+   :widths: 25 25 75
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Description
+   * - start_time
+     - Counter
+     - LC processing start timestamp (Unix epoch seconds)
+   * - end_time
+     - Counter
+     - LC processing end timestamp (Unix epoch seconds)
+   * - objects_scanned
+     - Counter
+     - Total objects scanned for lifecycle rules in current run
+   * - objects_pending
+     - Counter
+     - Objects currently pending lifecycle processing
+
+Lifecycle metrics are labeled counters tracked per-bucket in the ``rgw_lc_per_bucket`` section.
+
+Information about lifecycle metrics can be seen in the ``rgw_lc_per_bucket`` sections of the output of the ``counter schema`` command.
+
+To view lifecycle metrics in the Ceph Object Gateway go to the ``rgw_lc_per_bucket`` sections of the output of the ``counter dump`` command::
+
+    "rgw_lc_per_bucket": [
+        {
+            "labels": {
+                "bucket": "mybucket"
+            },
+            "counters": {
+                "start_time": 1728139406,
+                "end_time": 0,
+                "objects_scanned": 15000,
+                "objects_pending": 2300
+            }
+        },
+        ...
+    ]
+
+Lifecycle Counter Cache
+^^^^^^^^^^^^^^^^^^^^^^^
+
+To track lifecycle metrics by bucket the Ceph Object Gateway the config value ``rgw_lc_counters_cache`` must be set to ``true``. The default value is ``false``.
+
+This config value is set in Ceph via the command ``ceph config set client.rgw rgw_lc_counters_cache true``
+
+Since the lifecycle metrics are labeled perf counters, they live in memory. If the Ceph Object Gateway is restarted or crashes, all counters in the Ceph Object Gateway, whether in a cache or not, are lost.
+
+Lifecycle Counter Cache Size & Eviction
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``rgw_lc_counters_cache_size`` can be used to set number of entries in the cache.
```

Comment:
s/can/option can/

Also suggest wrapping in :confval:\`rgw_lc_counters_cache_size\`